packages/api/src/shared/helpers/FileHelper.test.ts

```diff
@@ -0,0 +1,37 @@
+import { detectCsvDelimiter } from "./FileHelper";
+
+describe("detectCsvDelimiter", () => {
+    it("should detect ';' as delimiter", () => {
```

Comment:
tu aurais pu utiliser `it.each` ici pour réduire le nombre de lignes, si jamais tu ne connais pas :)